docs/core/diagnostics/dotnet-counters.md

```diff
@@ -39,8 +39,53 @@ dotnet-counters [-h|--help] [--version] <command>
 
```

Comment:
Given the nature of the changes on this article, please update the ms.date as well on line 4 to 02/26/2020

Reply:
done! thanks for the tip!